components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -4183,4 +4187,12 @@ private void addUserAttributesToCache(SessionDataCacheEntry sessionDataCacheEntr
                 new DeviceAuthorizationGrantCacheEntry(sessionDataCacheEntry.getLoggedInUser().getUserAttributes());
         DeviceAuthorizationGrantCache.getInstance().addToCache(cacheKey, cacheEntry);
     }
+
+    private boolean isFapiConformant(String clientId) throws InvalidRequestException {
+        try {
```

Comment:
Add new line after method definition

Reply:
Fixed in 9ed19436be3c36b2fe703d66e088386ce1c78c86